corehq/apps/export/views/list.py

```diff
@@ -56,52 +56,52 @@
 from corehq.apps.export.views.utils import ExportsPermissionsManager, user_can_view_deid_exports
 
 
-class BaseExportListView(HQJSONResponseMixin, BaseProjectDataView):
-    template_name = 'export/export_list.html'
-    allow_bulk_export = True
+class ExportListHelper(object):
+    '''
+    Encapsulates behavior that varies based on model (form, case, or sms)
+    and is needed by the function-based views in this module.
+    '''
+    form_or_case = None         # None if this handles both forms and cases
     is_deid = False
 
-    lead_text = ugettext_lazy('''
-        Exports are a way to download data in a variety of formats (CSV, Excel, etc.)
-        for use in third-party data analysis tools.
-    ''')
-
-    @use_select2
-    @method_decorator(login_and_domain_required)
-    def dispatch(self, request, *args, **kwargs):
-        self.permissions = ExportsPermissionsManager(self.form_or_case, request.domain, request.couch_user)
-        self.permissions.access_list_exports_or_404(is_deid=self.is_deid)
-
-        return super(BaseExportListView, self).dispatch(request, *args, **kwargs)
+    @classmethod
+    def get(self, request, domain, form_or_case=None, is_daily_saved_export=False, is_feed=False, is_deid=False):
```

Comment:
You could cut down on some of these subclasses by passing `is_deid` to the constructor and setting it as a property.  The functionality doesn't differ that much.